commands/src/api/mod.rs

```diff
@@ -0,0 +1,71 @@
+use std::fmt::{Display, Formatter};
+use std::str::FromStr;
+
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+use uuid::Uuid;
+
+pub mod deployment;
+pub mod webrtc;
+
+/// Error types
+#[derive(Debug, Error)]
+pub enum Error {
+    #[error("Failed to deserialize command")]
+    DeserializeCommand,
+}
+
+/// API request packet type
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Request {
+    /// Request body
+    pub body: Body,
+    /// Response correlation string
+    pub respond_to: Option<String>,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+/// API request body
+pub enum Body {
+    /// Start pipeline deployment
+    StartDeployment(deployment::StartDeployment),
+    /// Stop deployment by id
+    StopDeployment(uuid::Uuid),
+    /// Discover cameras
+    Discover,
+    /// WebRTC subcommands collection
+    WebRtc(webrtc::Request),
+}
+
+impl FromStr for Request {
```

Comment:
Why do we use `FromStr` and `Display` for JSON (de)serialization here?

Reply:
Existing MQTT layer interface in API server and lumeod is based on strings
https://github.com/lumeohq/lumeod/blob/vyacheslavtroshin/ch961/refine-lumeod-api-server-protocol/src/api_client.rs#L121
https://github.com/lumeohq/lumeod/blob/7bad718788588f21442b0efe27b8845850afd85a/src/api_client.rs#L130

Reply:
Okay. Seems like something we should change separately.